sdk/kusto/arm-kusto/src/models/mappers.ts

```diff
@@ -493,7 +493,7 @@ export const DataConnection: msRest.CompositeMapper = {
       serializedName: "kind",
       clientName: "kind"
     },
-    uberParent: "DataConnection",
```

Comment:
With the right `uberParent` now the correct polymorphic child model will be picked up from the [discriminators map](https://github.com/Azure/azure-sdk-for-js/blob/6b526c0b7f44a399e80febab6bdbbdee79af6cc9/sdk/kusto/arm-kusto/src/models/mappers.ts#L1054) during [serialization](https://github.com/Azure/azure-sdk-for-js/blob/6b526c0b7f44a399e80febab6bdbbdee79af6cc9/sdk/core/core-http/lib/serializer.ts#L688)